package/CHANGELOG

```diff
@@ -13,18 +13,20 @@ The rules for this file:
   * release numbers follow "Semantic Versioning" http://semver.org
 
 ------------------------------------------------------------------------------
-mm/dd/17 richardjgowers, rathann, orbeckst
+mm/dd/17 richardjgowers, rathann, orbeckst, kain88-de, tyler.je.reddy
```

Comment:
Then we might need to change that throughout the CHANGELOG because that's the handle I found in there.